Sources/Widgets/Widgets3D/SplineWidget/index.js

```diff
@@ -62,6 +62,7 @@ const DEFAULT_VALUES = {
     key: 'Shift',
     status: 'down',
   },
+  defaultCursor: 'pointer',
```

Comment:
how about having that feature for ALL widgets ?

Reply:
Not all widgets change the cursor and some widgets might want some more complex systems than just defaultCursor/grabbingCursor (eg. IpmlicitPlaneWidget uses 4 different cursors).

Reply:
@floryst what do you think ?

Reply:
Exposing an API to modify the default/dragging/etc. cursors seems like a reasonable idea. However, if this API is added to the widget superclass, then there has to be some way of accommodating various cursor states (e.g. if a widget has special cursors for dragging edges vs corners, etc.) Right now I think it'll be easier for users who want to change cursor state to subclass the widget.

Also, do you want to add a `macro.setGet` in extend() for `defaultCursor`?

Reply:
Thank you @floryst .